packages/Webkul/Admin/src/Resources/views/settings/taxes/rates/index.blade.php

```diff
@@ -4,23 +4,139 @@
         @lang('admin::app.settings.taxes.rates.index.title')
     </x-slot:title>
 
-    <div class="flex justify-between items-center">
+    <div class="flex gap-4 justify-between items-center max-sm:flex-wrap">
         <p class="text-xl text-gray-800 dark:text-white font-bold">
             @lang('admin::app.settings.taxes.rates.index.title')
         </p>
-        
+
         <div class="flex gap-x-2.5 items-center">
-            <!-- Create New Pages Button -->
+            <!-- Tax Rate Emport -->
+            <v-tax-rates-import>
+                <button class="transparent-button hover:bg-gray-200 dark:hover:bg-gray-800 dark:text-white">
+                    @lang('admin::app.settings.taxes.rates.index.import.import-btn')
+                </button>
+            </v-tax-rates-import>
+
+            <!-- Tax Rate Export -->
+            <x-admin::datagrid.export src="{{ route('admin.settings.taxes.rates.index') }}"></x-admin::datagrid.export>
+
+            <!-- Create New Tax Rate Button -->
             @if (bouncer()->hasPermission('settings.taxes.tax-rates.create'))
-                <a 
-                    href="{{ route('admin.settings.taxes.rates.create') }}"
-                    class="primary-button"
-                >
+                <a href="{{ route('admin.settings.taxes.rates.create') }}" class="primary-button">
                     @lang('admin::app.settings.taxes.rates.index.button-title')
                 </a>
             @endif
         </div>
     </div>
-    
-    <x-admin::datagrid :src="route('admin.settings.taxes.rates.index')"></x-admin::datagrid>
+
+    <x-admin::datagrid 
+        :src="route('admin.settings.taxes.rates.index')" 
+        ref="datagrid"
+    >
+    </x-admin::datagrid>
+
+    @pushOnce('scripts')
+        <script type="text/x-template" id="v-tax-rates-import-template">
+            <div>
+                <!-- Tax Rate Import Button -->
+                <button
+                    href="{{ route('admin.settings.taxes.rates.import') }}"
+                    class="transparent-button hover:bg-gray-200 dark:hover:bg-gray-800 dark:text-white"
+                    @click="$refs.importTaxRates.toggle()"
+                >
+                    @lang('admin::app.settings.taxes.rates.index.import.import-btn')
+                </button>
+
+                <!-- Modal Form -->
+                <x-admin::form
+                    v-slot="{ meta, errors, handleSubmit }"
+                    as="div"
+                >
+                    <form
+                        @submit="handleSubmit($event, create)"
+                    >
```

Comment:
Remove the separations. Do it in a single line.

Reply:
Fixed